main.py

```
@@ -1,44 +1,113 @@
+        return
+    game = VIDEO_DATABASE[game_key]
+    buttons = []
+    for series in game["series"].keys():
+        buttons.append([InlineKeyboardButton(f"🎬 Серия {series}", callback_data=f"{game_key}|{series}")])
+    await update.message.reply_text(
+        f"🎮 **{game['name']}**",
+        reply_markup=InlineKeyboardMarkup(buttons),
+        parse_mode='Markdown'
+    )
+
+async def button_callback(update: Update, context: ContextTypes.DEFAULT_TYPE):
```

Ruff (0.15.1)

[warning] Unused function argument: `context`

(ARG001)